lib/datapacksexpand.js

```diff
@@ -962,6 +964,38 @@ DataPacksExpand.prototype.processDataPackData = function(dataPackType, parentNam
     }
 };
 
+DataPacksExpand.prototype.addDataPackToFullManifest = function(extension, dataPackType, expansionData) {
+    var self = this;
+
+    if (extension === 'scss'
+    && self.jobInfo.compileOnBuild 
+    && self.jobInfo.maxDepth != 0) {
+        var regex = new RegExp("(?:@import \")(.*?)(?:;)", "ig");
+        var found = expansionData.match(regex);
+        var templates = [];
+
+        for (var key in self.jobInfo.allDataSummary) {
```

Comment:
This isn't really needed. You could just get the templates from the regex then add them to fullManifest if fullManifest doesn't already contain the VlocityDataPackKey